src/backend/distributed/executor/multi_utility.c

```diff
@@ -816,6 +832,131 @@ ProcessAlterObjectSchemaStmt(AlterObjectSchemaStmt *alterObjectSchemaStmt,
 }
 
 
+/* TODO: Write function comments */
+static Node *
+ProcessVacuumStmt(VacuumStmt *vacuumStmt, const char *vacuumCommand, bool isTopLevel)
+{
+	Oid relationId = InvalidOid;
+	List *taskList = NIL;
+
+	if (vacuumStmt->relation == NULL)
+	{
+		return (Node *) vacuumStmt;
+	}
+
+	relationId = RangeVarGetRelid(vacuumStmt->relation, NoLock, false);
```

Comment:
In that case you'd have to be careful in followup code, which IIRC you weren't... Anyway, problem's gone now.